contracts/Package.sol

```diff
@@ -0,0 +1,45 @@
+pragma solidity 0.5.0;
```

Comment:
You can target higher version.

Reply:
Actually Truffle Framework v5.0.12 ships with solc targeting 0.5.0:
```
$ npx truffle version
Truffle v5.0.12 (core: 5.0.12)
Solidity v0.5.0 (solc-js)
Node v11.14.0
Web3.js v1.0.0-beta.37
```